apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Tests/Logging/GoogleLoggerTest.cs

```diff
@@ -359,6 +360,19 @@ public void Log_Labels()
             mockConsumer.VerifyAll();
         }
 
+        [Fact]
+        public void Log_DoesNotLogIfNullLabels()
```

Comment:
Wondering if there should be a test with a `RetryOptions.ExceptionHandling` value different than  `ExceptionHandling.Ignore` where the exception would propagated.